e2e/framework/testserver.go

```diff
@@ -351,17 +352,32 @@ func (t *TestServer) Start(ctx context.Context) error {
 	return err
 }
 
+// SignTx is a helper method for signing transactions
+func (t *TestServer) SignTx(
+	transaction *types.Transaction,
+	privateKey *ecdsa.PrivateKey,
+) (*types.Transaction, error) {
+	return t.Config.Signer.SignTx(transaction, privateKey)
+}
+
 // DeployContract deploys a contract with account 0 and returns the address
-func (t *TestServer) DeployContract(ctx context.Context, binary string) (web3.Address, error) {
+func (t *TestServer) DeployContract(
+	ctx context.Context,
+	binary string,
+	privateKey *ecdsa.PrivateKey,
+	sender types.Address,
```

Comment:
Is is possible to calculate `sender` from `privateKey`?

Reply:
Extracted a helper method that any module can now use 👍 

9080129